### DISCUSSION TITLE
Are we need a primitive type for binary data in credential ?

### DISCUSSION THREAD

**Discussion by rssh** [zakaio/atala-prism-schema#3] [Polls]
Are we need binary data in credentials?
  One of the use-cases: is biometric data, such as voice characteristics.

How to represent one in json [?] - base64?

**Comment by rodolfomiranda** [comment]
In think BASE64 is the most common representation of binary data. Last month was [Identiverse](https://identiverse.com/idv2022) conference and biometrics was a main topic. We should check if there are some definition around mapping bio info  into credential.
